nservicebus/nhibernate/outbox_customizing_nhibernate_[7,).partial.md

```diff
@@ -0,0 +1,15 @@
+## Customizing outbox record peristence
+
+By default the outbox records are persisted in the following way:
+ * The table has an auto-incremented integer primary key
+ * The `MessageId` column has a unique index
+ * There are indices on `Dispatched` and `DispatchedAt` columns.
+
+Following API can be used to provide a different mapping of outbox data to the underlying storage:
+
+snippet:OutboxNHibernateCustomMapping
+
+The following properties of the original mapping need to be preserved when providing a custom mapping:
```

Comment:
Think it reads better if we invert it, emphasising on when this needs to happen:

>Should custom mapping be needed, the following characteristics of the original mapping needs to be preserved: